.github/workflows/maven.yaml

```diff
@@ -48,7 +48,7 @@ jobs:
         key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
         restore-keys: ${{ runner.os }}-m2
     - name: Build with Maven
-      run: mvn -B -V -e -ntp "-Dstyle.color=always" verify javadoc:jar
+      run: mvn -B -V -e -ntp "-Dstyle.color=always" -DskipITs verify javadoc:jar
```

Comment:
It would be nice to get the ITs running reliably in GitHub Actions. More could be done here to split the basic build from the IT runs, but could be a follow-on task.

Reply:
Maybe.  The ITs were taking like 40-50 mins to run in CI, where it takes 8 to 10 mins on my machine.  Also they are flaky.  Not sure CI should take that long.